wizeline/falcon/middlewares/bodyParser.py

```diff
@@ -0,0 +1,58 @@
+import json
+from json.decoder import JSONDecodeError
+
+import falcon
+
+class BodyParserMiddleware:
+    def process_resource(self, req, resp, resource, params):
+        if self._is_middleware_enabled(resource) and self._request_method_has_payload(req):
+            if self._is_json_content_type(req):
+                try:
+                    req.text = self._get_payload(req)
+
+                    if req.text.strip() != '':
```

Comment:
What are we validating with this?

Reply:
Just to avoid trying to parse white spaces strings
```py
JSON.loads('            ') # this raises a JSONDecodeError
```

Reply:
You should abstract that complex line into a more descriptive method like:
```python
if self._is_empty(req.text):
  # do stuff
...

def is_empty(self, text):
  return text.strip() != ''
```